src/data/migrations/20200506004924-add-route-name.js

```diff
@@ -0,0 +1,17 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.addColumn('Routings', 'name', {
+      type: Sequelize.TEXT,
+      allowNull: false,
+      defaultValue: 'route'
```

Comment:
unique index shouldn't have default value imo

Reply:
It does not let me do `allowNull: false` without a default value :S But I agree with you